### ISSUE TITLE
Refine Net

### ISSUE THREAD

**Issue by Tiamo666** [GengDavid/pytorch-cpn#4]
I guess maybe there is some problems in implementation of refine net. 
In your refineNet.py, you define the forward pass as follows: 
def forward(self, x): 
        refine_fms = []
        for i in range(4):
            refine_fms.append(self.cascade[i](x[i]))
        out = torch.cat(refine_fms, dim=1)
        out = self.final_predict(out)                                           
        return out 
I think you should inverse the x, eg: x = x[-1:], because x[0] is the smallest feature map, and x[3] is biggest feature map. And there are 3 bottlenecks after smallest feature map , 0 bottleneck after biggest feature map according to paper.

**Comment by GengDavid** [comment]
Hi, @Tiamo666 
Yes, you are right. It is suppose to have 3 bottlenecks after the smallest feature map and no bottleneck after the biggest feature map. 
Thanks for pointing it out! 
I'll correct it.

**Comment by Tiamo666** [comment]
Thanks a lot for your reimplementation work.
